tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -312,12 +327,75 @@ private static void StartDiagnosticManager()
         }
 #endif
 
-        internal static void InitializeLiveDebugger()
+        private static void InitAsyncMethods(Tracer tracer)
+        {
+            Task.Run(
+                async () =>
+                {
+                    var discoveryService = await InitializeDiscoveryService(tracer.Settings.Exporter).ConfigureAwait(false);
+
+                    var products = new List<Product>() { LiveDebugger.Product };
```

Comment:
If `products` will always be hard-coded, why not making it an array?
This way you'll eliminate confusion down the path of moving it from one place to another as seeing `List` suggests it's mutable while it's not designed to be.